app/page.tsx

```diff
@@ -2,35 +2,41 @@ import WelcomeTexts from '@/components/welcome-texts/WelcomeTexts';
 import PostFeed from '@/components/post-feed/PostFeed';
 import { PostEditor } from '@/components/post-editor/PostEditor';
 import { auth } from '@/app/api/auth/[...nextauth]/auth';
-import { PostEditorPlaceholder } from '@/components/placeholders/PostEditorPlaceholder';
 import { getMumblePosts } from '@/utils/helpers/posts/getMumblePosts';
+import PostsLoader from '@/components/posts-loader/PostsLoader';
+import frontendConfig from '@/config';
 
 export default async function Page() {
   const session = await auth();
-  const feedData = await getMumblePosts({});
+  const feedData = await getMumblePosts({
+    offset: 0,
+    limit: frontendConfig.feed.defaultAmount,
+  });
   return (
     <div className="mx-auto bg-slate-100 pt-8">
       <div className="max-w-4xl mx-auto py-8">
         <WelcomeTexts
           title="Welcome to Mumble"
-          description="Did you hear that? They've shut down the main reactor. We'll be destroyed for sure. This is madness! We're doomed! There'll be no escape for the Princess this time. What's that? Artoo! Artoo-Detoo, where are you? At last! Where have you been? They're heading in this direction."
+          description="Did you hear that? They've shut down the main reactor."
         />
       </div>
       <div className="max-w-4xl mx-auto">
         {session && (
           <div>
-            <PostEditorPlaceholder />
             <PostEditor isFeedPage={true} />
           </div>
         )}
 
         <div className="max-w-4xl mr-auto ml-auto">
+          {/*server side rendered*/}
```

Comment:
Do we need these comments if those components are client / server side rendered? If someone change those component's rendering strategy, those comments could be irritating.